kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/resources/vault/KubeVaultTestKmsFacade.java

```diff
@@ -126,11 +133,50 @@ protected URI getVaultUrl() {
         return URI.create("http://" + vault.getVaultUrl());
     }
 
+    /**
+     * Gets vault version.
+     *
+     * @return the vault version
+     */
+    public String getVaultVersion() {
+        return vault.getVersionInstalled();
+    }
+
     @Override
     public TestKekManager getTestKekManager() {
         return new VaultTestKekManager();
     }
 
+    /**
+     * Is correct version installed boolean.
+     *
+     * @return the boolean
+     */
+    public boolean isCorrectVersionInstalled() {
+        String installedVersion = getVaultVersion();
+        String expectedVersion = VaultTestKmsFacade.HASHICORP_VAULT.split(":")[1];
```

Comment:
with the change above, you can use DockerImageName#getVersionPart() and get rid of the split.